miner/celo_defaults.go

```diff
@@ -0,0 +1,25 @@
+package miner
+
+import (
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/params"
+)
+
+// cStables addresses on mainnet
+var (
+	cUSD_TOKEN  = common.HexToAddress("0x765DE816845861e75A25fCA122bb6898B8B1282a")
+	cEUR_TOKEN  = common.HexToAddress("0xD8763CBa276a3738E6DE85b4b3bF5FDed6D6cA73")
+	cREAL_TOKEN = common.HexToAddress("0xe8537a3d056DA446677B9E9d6c5dB704EaAb4787")
```

Comment:
Should we use the chance to add USDT and USDC as well?

Reply:
What would be sensible block-limit defaults for them? 0.9 in analogy to `cUSD`?

Reply:
Yes, I think `0.9` would do.